packages/ui/src/composites/wui-account-button/index.ts

```diff
@@ -0,0 +1,54 @@
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import '../wui-avatar'
+import '../../layout/wui-flex'
+import '../../components/wui-text'
+import '../../components/wui-image'
+import { elementStyles, resetStyles } from '../../utils/ThemeUtil'
+import { UiHelperUtil } from '../../utils/UiHelperUtils'
+import styles from './styles'
+
+@customElement('wui-account-button')
+export class WuiAccountButton extends LitElement {
+  public static styles = [resetStyles, elementStyles, styles]
+
+  // -- State & Properties -------------------------------- //
+  @property() public imageSrc?: string = undefined
```

Comment:
We need `networkSrc` and `avatarSrc` props (in case user has ens profile, otherwise we show wui-avatar)